react-app/src/features/forms/waiver/capitated/Amendment.tsx

```diff
@@ -9,123 +9,124 @@ import {
   Input,
   RequiredIndicator,
 } from "@/components";
-import { Link } from "react-router-dom";
 import { formSchemas } from "@/formSchemas";
 import { FAQ_TAB } from "@/router";
+import { Link } from "react-router-dom";
 import { getFAQLinkForAttachments } from "../../faqLinks";
 
-export const AmendmentForm = () => (
-  <ActionForm
-    schema={formSchemas["capitated-amendment"]}
-    title="1915(b) Comprehensive (Capitated) Waiver Amendment Details"
-    breadcrumbText="1915(b) Comprehensive (Capitated) Waiver Amendment"
-    fields={({ control }) => (
-      <>
-        <div className="flex flex-col">
-          <FormLabel className="font-semibold" htmlFor="1975b">
-            Waiver Authority
-          </FormLabel>
-          <span className="text-lg font-thin" id="1975b">
-            1915(b)
-          </span>
-        </div>
-        <FormField
-          control={control}
-          name="waiverNumber"
-          render={({ field }) => (
-            <FormItem>
-              <FormLabel
-                className="font-semibold"
-                data-testid="existing-waiver-label"
-              >
-                Existing Waiver Number to Amend <RequiredIndicator />
-              </FormLabel>
-              <p className="text-gray-500 font-light">
-                Enter the existing waiver number you are seeking to amend in the
-                format it was approved, using a dash after the two character
-                state abbreviation.
-              </p>
-              <FormControl className="max-w-sm">
-                <Input
-                  ref={field.ref}
-                  value={field.value}
-                  onChange={(e) =>
-                    field.onChange(e.currentTarget.value.toUpperCase())
-                  }
-                />
-              </FormControl>
-              <FormMessage />
-            </FormItem>
+interface AmendmentFormProps {
+  waiverId?: string;
+}
+
+export const AmendmentForm = ({ waiverId = "" }: AmendmentFormProps) => {
```

Comment:
What's the point of having a default `waiverId` value? Is it to serve as a default value for the `defaultValues` prop? Wouldn't it be clearer to set the fallback string there?

Reply:
It's so by default we don't render the existing waiver number:
```
{waiverId !== "" ? (
    <div>
      <p className="font-semibold">Existing Waiver Number to Amend</p>
      <p className="text-lg font-thin">{waiverId}</p>
    </div>
  )
  ```
Open to changing it though. We could do something like `waiverId.length > 0` but that doesn't read as well.

Reply:
yea, I can update if that's the preference

Reply:
I'll leave it to you, but in terms of explicitness, I think `undefined` would be easier to reason about than an empty string since it's more explicit